package.json

```diff
@@ -8,13 +8,14 @@
     "build": "babel . -d build --copy-files --ignore node_modules,postman,build",
     "init-db": "babel-node src/db/init.js",
     "lint": "eslint index.js src/",
-    "test": "mocha --reporter spec src/**/*.test.js --require babel-register"
+    "test": "find ./src -name '*tests.js' | xargs mocha --require babel-register"
```

Comment:
Or I can use cygwin terminal and we won't have any problems.
![image](https://user-images.githubusercontent.com/17383790/28502410-a6e73e30-6ffa-11e7-92ff-310be6eda458.png)